curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.md

```diff
@@ -28,7 +28,7 @@ var ourStr = "I come first. " + "I come second.";
 The string `I come first. I come second.` would be displayed in the console.
 # --instructions--
 
-Build `myStr` from the strings `This is the start. ` and `This is the end.` using the `+` operator.
+Build `myStr` from the strings <pre>`This is the start. `</pre> and `This is the end.` using the `+` operator.
```

Comment:
This still doesn't work, it appears as before

Maybe it's actually one of the crowdin workflows that needs to be used for this 🤷

Reply:
Yeah, I'm stumped myself. Please let me know if you have any ideas. I can try to jump into the crowdin docs and get familiar with it, if needed.

Reply:
@nhcarrigan is the Crowdin expert, maybe he have some ideas

Reply:
Honestly might be better to reword the challenge a bit.

Reply:
maybe adding quotes?
I know we were removing those from strings but maybe here it's best for translating

Reply:
Or:
> "Build `myStr` from the strings `This is the start.` and `This is the end.` using the `+` operator. Be sure to include a space between the two strings.